recipes/sentencepiece/build.sh

```diff
@@ -0,0 +1,12 @@
+mkdir build
+cd build
+cmake
```

Comment:
This is missing a backslash
```suggestion
cmake \
```